platform/linux-generic/include/odp_internal.h

```diff
@@ -55,10 +56,16 @@ struct odp_global_data_s {
 	odp_cpumask_t control_cpus;
 	odp_cpumask_t worker_cpus;
 	int num_cpus_installed;
+	config_t *libconfig_default;
+	config_t *libconfig_runtime;
+	config_t libconfig_default_structure;
+	config_t libconfig_runtime_structure;
```

Comment:
Can we please have just two config structures here (default and running)? I wonder, if we can actually merge default with provided configurations into a single config structure?

Reply:
Merging is not required in the first pass, but it might be a good thing in future.

Reply:
Yeah, I actually tried merging the files but it ended up being painful since libconfig doesn't provide good functions for doing that. So I went with the "safe" solution. I'll try if I can at least get rid of the extra pointers.